assets/javascript/app.js

```diff
@@ -1,4 +1,43 @@
 //= require_directory ./vendor
 //
 
-// custom JS can go in this file, or another, then you add another require line
\ No newline at end of file
+// custom JS can go in this file, or another, then you add another require line
+
+function showCareerPrev(currIndex) {
+  'use strict';
+
+  var newIndex = parseInt(currIndex) - 1;
```

Comment:
I'm guessing you tried this, but line 6 of ` templates/partials/career.mustache` makes me think that you shouldn't need to do the `parseInt` because the parameters won't be passed in as strings. A quick look at the web page seems to confirm this.